packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -754,6 +754,27 @@ Specifies a set of Groups whose Users are to be included or excluded
     }
   }
 ```
+#### UserType Condition object
+
+Specifies a set of UserTypes are to be included or excluded. You can use the [UserTypes API](/docs/reference/api/user-types/) to manage UserTypes.
```

Comment:
"Specifies which User Types to include and/or exclude"

And "User Types" should link to further information about User Types.